src/index.ts

```diff
@@ -54,6 +66,9 @@ generator client {
     model.generateSchema();
     prismaSchema += model.prismaModel;
   }
+  for (const _enum of enums) {
```

Comment:
It's better to have enums above model definitions. Could you move loop above?

Reply:
We can see it in refactoring the code.